packages/eslint-plugin/src/rules/no-duplicate-enum-values.ts

```diff
@@ -9,7 +9,7 @@ export default util.createRule({
       description: 'Disallow duplicate enum member values',
       recommended: 'strict',
     },
-    hasSuggestions: true,
+    hasSuggestions: false,
```

Comment:
Many of these rules indicated they had suggestions but do not actually provide suggestions.